src/overlap_check/overlap.cpp

```diff
@@ -108,7 +108,6 @@ check_instance_for_overlaps(std::shared_ptr<Interface> MBI,
 
       EntityHandle vert = all_verts[i];
       CartVect loc;
-      rval = MBI->get_coords(&vert, 1, loc.array());
```

Comment:
Although the value of `rval` is not checked, this call is necessary to populate `loc` correctly.